.github/workflows/backend-production-deploy.yml

```diff
@@ -19,7 +22,6 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0 # fetch history for all branches and tags
-          token: ${{ secrets.GH_TOKEN }} # use PAT with permissions to push to master
```

Comment:
I'm curious why the other workflow using the auto generated `GITHUB_TOKEN` secret, while this one uses nothing.

Reply:
Where is this other code?

Reply:
https://github.com/dandi/dandi-archive/blob/796a6ae83749b66a2fae422b58533f6d4a84ed09/.github/workflows/release.yml#L28